### PR TITLE
fixed word spacing, spread out additionalSpacing rather than putting it ...

### PR DESCRIPTION
**Pull request by coax75ohm**
...all after the first word on line

### CODE REVIEW

File: src/org/albite/book/view/TextPage.java [hunk 1]
```diff
@@ -605,7 +605,7 @@ private void positionWordsOnLine(
                 spacing = wordSpacing;
             } else {
                 /* calculate spacing so words would be justified */
-                if (words.size() > 1) {
+                if (wordsSize > 1) {
```
Comment (coax75ohm) [inline src/org/albite/book/view/TextPage.java]: Looking at it again, would be better to make it (wordsSize1 > 0)

**Comment by coax75ohm** [comment]
Sorry about the extra commits, only meant for the first one (7f89712) to be here. I'll start a new branch for further edits. Is there a way to change what branch this issue points to?

**Comment by dumbledore** [comment]
I suggest that you fork the project and push the commits there.... I won't be able to review your patches in the next couple of months :-/

Glad you cared though. I'm touched! :-)

**Comment by coax75ohm** [comment]
All right, just wanted to share that one commit here since it's a general typography fix (and it's very short so there's not much to review ;). The rest of my fork is aimed at a specific platform so you wouldn't want those changes anyway.

I'll close this since you've seen it.

**Pull request closed without merging**